Forge.TreeWalker.UnitTests/test/TreeSchemaValidatorTests.cs

```diff
@@ -0,0 +1,175 @@
+//-----------------------------------------------------------------------
+// <copyright file="TreeSchemaValidatorTests.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// <summary>
+//     Tests for the TreeSchemaValidator class.
+// </summary>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Forge.TreeWalker.UnitTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Microsoft.Forge.DataContracts;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Schema;
+    using Microsoft.Forge.TreeWalker;
+
+    [TestClass]
+    public class TreeSchemaValidatorTests
+    {
+        private string forgeTreeAsString;
+        private ForgeTree treeSchema;
+        private Dictionary<string, ForgeTree> treeSchemas;
+        private string forgeTreeFromPath;
+        private string schemaDirectoryPath;
+        private string stringRules;
+        private string rulesForDictionary;
+        private JSchema linkedRulesForDictionary;
+        private JSchema jschemaRules;
+        private string invalidSchemaDirectoryPath;
+        private string invalidSchemaWithErrorContent;
+        private string forgeTreeDictionaryAsString;
```

Comment:
Once you re-order the variable declarations inside TestInitialize, define the fields in the same order here. You don't need to add newlines between any fields here though.